src/base/utils/misc.cpp

```diff
@@ -503,9 +503,9 @@ QList<bool> Utils::Misc::boolListfromStringList(const QStringList &l)
 
 bool Utils::Misc::isUrl(const QString &s)
 {
-    const QString scheme = QUrl(s).scheme();
-    QRegExp is_url("http[s]?|ftp", Qt::CaseInsensitive);
-    return is_url.exactMatch(scheme);
+    static const QRegExp isURL("http[s]?|ftp", Qt::CaseInsensitive);
```

Comment:
@evsh, your memory is strange... 
>For a starters, fix coding style.

Don't forget that when you publish your own PR. In addition, the adherence of coding style is not something external, but an integral part of each coding job in the project.
As for this place, I rewrote it (because it is faster than arguing here with you).
>If you are concerned about performance you might want to use QRegularExpression instead of making it static.

@Chocobo1, I leave it as static const. I don't see any reason why it shouldn't be static. What do you think?